### DISCUSSION TITLE
Tab cycle though options

### DISCUSSION THREAD

**Discussion by huyhoang8398** [davatorium/rofi#1645] [Q&A] [answered]
Is this possible to use tap to cycle throught options
i.e: im in the last options and i want to jump back to the first options using tap

**Comment by DaveDavenport** [comment, marked as answer]
should be default behaviour. You can change this in a setting on the listview.

**Comment by huyhoang8398** [comment]
I didnt know that there is an option cycle in list view, thanks

**Reply by DaveDavenport** [reply]
its in the manpage(s).

**Reply by huyhoang8398** [reply]
Thanks it works perfect now

**Comment by huyhoang8398** [comment]
```css
listview {
    ...
    cycle:  true;
    ...
}
```
